projects/admin/src/app/record/detail-view/entities-detail-view/local/entities-local-detail-view.component.ts

```diff
@@ -37,14 +38,24 @@ export class EntitiesLocalDetailViewComponent implements DetailRecord {
   /** Enum of type of Entity */
   entityType = EntityType;
 
+    /**
+   * Is operation log enabled
+   * @return boolean
+   */
+    get isEnabledOperationLog(): boolean {
+      return this._operationLogsService.isLogVisible('local_entities');
+    }
```

Comment:
It seems getter methods is called each time something change on component model... and it happens a lot during component init XD. It should be more robust to create a component variable `operationLogEnabled: bool` and set the correct value during the `ngOnInit` ?

Reply:
right 👍🏻